packages/vuetify/src/mixins/overlayable.ts

```diff
@@ -211,7 +211,19 @@ export default Vue.extend<Vue & Toggleable & Stackable & options>().extend({
       if (this.$vuetify.breakpoint.smAndDown) {
         document.documentElement!.classList.add('overflow-y-hidden')
       } else {
-        window.addEventListener('wheel', this.scrollListener as EventHandlerNonNull, { passive: false })
+        // IE11 does not support passive
+        let supportsPassive = false
+        try {
+          const opts = Object.defineProperty({}, 'passive', {
+            get: () => {
+              supportsPassive = true
+            }
+          })
+          window.addEventListener('testPassive', () => {}, opts)
+          window.removeEventListener('testPassive', () => {}, opts)
```

Comment:
This line does nothing. There's an example of this on MDN: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Safely_detecting_option_support

Reply:
Having `supportPassive = false` in the catch block seems unnecessary since the var is initialized as false, but I guess it doesn't hurt anything either.